src/components/FileRepo.js

```diff
@@ -180,6 +188,7 @@ const AggregationsWrapper = compose(injectState, withTheme)(
                 <AdvancedFacetViewModalContent
                   {...{
                     ...props,
+                    translateSQONValue,
```

Comment:
technically this and the `translateSQONValue` below would be in `props` before so this extraction isn't necessary, but if it's here for readability, that's fine for me too :)